aten/src/ATen/native/quantized/cpu/qconv_unpack.cpp

```diff
@@ -146,6 +146,10 @@ static auto registry =
             " -> (Tensor unpacked_weights, Tensor? B_origin)",
             c10::RegisterOperators::options().kernel<QConvUnpackWeightsInt8<2>>(
                 TensorTypeId::CPUTensorId))
+        .op("quantized::conv2d_unpack(Tensor packed_weights)"
```

Comment:
can you add a comment on why we bind both and that one is deprecated? same for the _pack